make error

/home/hzx/shit/g2o/g2o/core/optimizable_graph.cpp: In member function ‘bool g2o::OptimizableGraph::verifyInformationMatrices(bool) const’:
/home/hzx/shit/g2o/g2o/core/optimizable_graph.cpp:811:25: error: ‘join’ is not a member of ‘fmt’
  811 |                    fmt::join(ids, " "));
      |                         ^~~~
/home/hzx/shit/g2o/g2o/stuff/logger.h:97:28: note: in definition of macro ‘G2O_WARN’
   97 |       spdlog::level::warn, __VA_ARGS__)
      |                            ^~~~~~~~~~~
/home/hzx/shit/g2o/g2o/core/optimizable_graph.cpp:814:25: error: ‘join’ is not a member of ‘fmt’
  814 |                    fmt::join(ids, " "));
      |                         ^~~~
/home/hzx/shit/g2o/g2o/stuff/logger.h:97:28: note: in definition of macro ‘G2O_WARN’
   97 |       spdlog::level::warn, __VA_ARGS__)
      |                            ^~~~~~~~~~~
make[2]: *** [g2o/core/CMakeFiles/core.dir/build.make:174：g2o/core/CMakeFiles/core.dir/optimizable_graph.cpp.o] 错误 1
make[1]: *** [CMakeFiles/Makefile2:1138：g2o/core/CMakeFiles/core.dir/all] 错误 2
make: *** [Makefile:156：all] 错误 2

I have installed fmt, but there is a problem.
Can someone tell me the solution?

--- a/g2o/core/optimizable_graph.cpp
+++ b/g2o/core/optimizable_graph.cpp
@@ -808,10 +808,10 @@ bool OptimizableGraph::verifyInformationMatrices(bool verbose) const {
           ids[i] = e->vertex(i)->id();
         if (!isSymmetric)
           G2O_WARN("Information Matrix for an edge is not symmetric: {}",
-                   fmt::join(ids, " "));
+                   strJoin(ids.begin(), ids.end(), " "));
         else
           G2O_WARN("Information Matrix for an edge is not SPD: {}",
-                   fmt::join(ids, " "));
+                   strJoin(ids.begin(), ids.end(), " "));
         if (isSymmetric)
           G2O_WARN("eigenvalues: {}", eigenSolver.eigenvalues().transpose());
       }

